crate/operator/change_compute.py

```diff
@@ -142,8 +179,30 @@ def generate_body_patch(
     That patch modifies cpu/memory requests/limits based on compute_change_data.
     It also patches affinity as needed based on the existence or not of requests data.
     """
+
+    crd = await get_cratedb_resource(namespace, name)
```

Comment:
we could avoid this additional request by passing the `body: kopf.Body` down [here](https://github.com/crate/crate-operator/blob/master/crate/operator/change_compute.py#L58)?